transcribe/scribe_modules/k8s_configmaps.py

```diff
@@ -23,9 +23,24 @@ def __iter__(self):
             yield attr, value
 
     def _parse(self, items_full):
-        # No modification is needed to output. Note: some of the configmap
-        # data comes in as yaml files. For example see kubelet in stockpile_configmap
+        # Flatten some of the dictionaries to lists
+        def _to_list(first,second,my_dict):
```

Comment:
This function is being used by other modules as well so probably makes sense to include it in [utils](https://github.com/cloud-bulldozer/scribe/blob/master/transcribe/scribe_modules/lib/util.py) similar to the existing function in there ( format_url ) and import it similar to how yum_repos module does it here https://github.com/cloud-bulldozer/scribe/blob/master/transcribe/scribe_modules/yum_repos.py

Reply:
agreed. Updated